src/entity-schema/EntitySchema.ts

```diff
@@ -247,4 +247,22 @@ export interface EntitySchema { // todo: make it-to-date
         };
     };
 
+     /**
+     * Entity indices options.
+     */
+    indices: {
+        [indexName: string]: {
+
+            /**
+             * Index column names.
+             */
+            columns: string[];
+
+            /**
+             * Index uniqueness.
+             */
+            unique: boolean;
```

Comment:
can you please add `sparse` option as well? (it is used in mongodb)

Reply:
Done